card/mod/standard/set/all/rich_html/menu.rb

```diff
@@ -22,7 +22,8 @@
   end
 
   def menu_path_opts
-    opts = { slot: { home_view: voo.home_view } }
+    opts = { slot: { home_view: (voo.home_view || @slot_view),
```

Comment:
this solution is ok, I guess, but the question for me is how we're getting here without a home view.  My first guess is that is one of the things that gets taken care of when we do `nest cardish, view: viewname` but NOT when we do `subformat(cardish).render_viewname`

Also, I'm a little surprised this isn't taken care of by the slotter handling...